src/metadata/metadata_handler.cc

```diff
@@ -234,7 +234,7 @@ std::string MetadataHandler::mapContentHandler2String(int ch)
     return "Unknown";
 }
 
-metadata_fields_t MetadataHandler::remapMetaDataField(const std::string& fieldName)
+metadata_fields_t MetadataHandler::remapMetaDataField(std::string_view fieldName)
```

Comment:
No reference?

Reply:
AFAIK string_view is trivially copyable.